share/spack/gitlab/cloud_pipelines/stacks/ml-linux-ppc64le-cpu/spack.yaml

```diff
@@ -0,0 +1,99 @@
+spack:
+  view: false
+  packages:
+    all:
+      require: target=ppc64le
+      variants: ~cuda~rocm
+
+  specs:
+    # Horovod
+    # See pytorch failures
+    #- py-horovod tensor_ops=mpi controllers=mpi frameworks=pytorch
+
+    # Hugging Face
+    - py-transformers
+
+    # JAX
+    - py-jax
+    - py-jaxlib
+
+    # Keras
+    # Fails to install bazel and py-tensorboard-data-server
+    #- py-keras
+    - py-keras-applications
+    - py-keras-preprocessing
+    - py-keras2onnx
+
+    # MXNet
+    - mxnet
+
+    # PyTorch
+    # Fails in CI when running `cmake --build . --target install --config Release -- -j 12`
+    # with output: `Preparing metadata (pyproject.toml): finished with status 'error'`
```

Comment:
This is a generic error that means "something went wrong". The real error is farther up in the output. If you share a log with me I can take a look.